db/migrate/20150416231106_add_source_to_webhook.rb

```diff
@@ -0,0 +1,6 @@
+class AddSourceToWebhook < ActiveRecord::Migration
+  def change
+    add_column :webhooks, :source, :string, default: 'any_ci'
+    Webhook.update_all source: 'any_ci'
```

Comment:
My understanding is default is for new records, update_all is for existing ones in our db.